### DISCUSSION TITLE
How Do I Reset the Generals/Divisionals?

### DISCUSSION THREAD

**Discussion by Aduragbemi-Adeyinka** [GrandOrgue/grandorgue#1827] [Help] [answered]
Hello Everyone, 

I want to reset the registration pistons, to a "never used before" state. How will I do that? 

I tried doing a "null registration" on them, that is, cancel all stops then store the null registration on the pistons. But now those pistons recall a null registration, whereas a piston that has never been used before will not cause any registration change when pressed.

**Comment by oleg68** [comment, marked as answer]
Export combinations, edit the yaml file and import it.

**Reply by Aduragbemi-Adeyinka** [reply]
OK. Thanks @oleg68. I know how to export and import, but I don't know how to edit.

**Reply by oleg68** [reply]
Open the .yaml file with your favorite text editor and remove unnecessary entries.

**Reply by Aduragbemi-Adeyinka** [reply]
Thank you very much.
